alacritty_terminal/src/term/mod.rs

```diff
@@ -1586,7 +1591,30 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn move_forward_tabs(&mut self, count: u16) {
-        trace!("[unimplemented] Moving forward {} tabs", count);
+        trace!("Moving forward {} tabs", count);
+        self.damage_cursor();
+
+        let num_cols = self.columns();
+        let old_col = self.grid.cursor.point.column.0;
+        for _ in 0..count {
+            let mut col = self.grid.cursor.point.column;
+
+            if col == num_cols - 1 {
+                break;
+            }
+
+            for i in col.0 + 1..num_cols {
+                col = index::Column(i);
+                if self.tabs[col] {
+                    break;
+                }
+            }
```

Comment:
Could this be rewritten as `self.tabs[col..].iter().find(|tab| tab).unwrap_or(num_cols - 1)`? It feels like that should work, but I'm not sure.

Reply:
it's way more code than what I've wrote and harder to read. You need to remember that your slice stuff is _relative_, but you need absolute, and then your unwrap must be converted to relative to then be converted to absolute, not saying that you have to add trait impl to even get an index. Just not worth it.

Reply:
Fair enough.